views/books.pug

```diff
@@ -0,0 +1,5 @@
+extends layout
```

Comment:
Would be great to see if we could nest books.pug in its own layout and call layout from within.